config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_logginglogmetrics.logging.cnrm.cloud.google.com.yaml

```diff
@@ -324,29 +324,28 @@ spec:
                     - external
                   required:
                   - name
-                  - kind
                 - not:
                     anyOf:
                     - required:
                       - name
                     - required:
                       - namespace
-                    - required:
-                      - kind
                   required:
                   - external
                 properties:
                   external:
-                    description: The external name of the referenced resource
+                    description: The `projectID` field of a project, when not managed
```

Comment:
We should not mention "KCC" in our CRD: this is not the approved branding name we should use.
Also, the field name is not `projectID` but `projectId` (in the API or gcloud) or `ID` (in the UI).

I think we should add checks in the CRD field description to avoid the mentioning of `KCC`.